tools/train.py

```diff
@@ -39,6 +40,7 @@ def get_args() -> Namespace:
     parser = ArgumentParser()
     parser.add_argument("--model", type=str, default="padim", help="Name of the algorithm to train/test")
     parser.add_argument("--model_config_path", type=str, required=False, help="Path to a model config file")
+    parser.add_argument("--log-level", type=str, help="<DEBUG, INFO, WARNING, ERROR>")
```

Comment:
Is this `log-level` passed to the console logger somewhere?

Reply:
Not yet, it's a bit more involved than we thought. It's something I was going to bring up. This could be either addressed in a separate PR, or this one needs to be modified a little bit.